bogota-july-2018/JScalc/calculator.js

```diff
@@ -0,0 +1,13 @@
+var display = document.getElementById("result"); 
+
+function getInput(data){
+  display.innerHTML += data;
+}
+
+function displayResult(){
+  display.innerHTML = eval(display.innerHTML);
```

Comment:
This is a quite simple approach, and probably it works, but eval has several pitfalls, and the exercise said "basic calculator" those ones don't display the operation. So that requirement is not ok ;)